lib/sycamore/sycamore/materialize.py

```diff
@@ -539,6 +539,34 @@ def local_source(self) -> list[Document]:
 
         return ret
 
+    def load_metadata(self) -> list[MetadataDocument]:
+        self._verify_has_files()
+        if not self._fshelper.file_exists(self._success_path()):
+            logging.warning(f"materialize.success not found in {self._orig_path}. Returning partial data")
+        from sycamore.utils.sycamore_logger import LoggerFilter
+
+        limited_logger = logging.getLogger(__name__ + ".limited_local_source")
```

Comment:
I'd name this something like .load_metadata; since the primary use would be to control that logger separately from other ones.